src/logic/operate.js

```diff
@@ -0,0 +1,26 @@
+const Big = require('big.js');
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    try {
+      return one.div(two).toString();
+    } catch (err) {
+      return "Can't divide by 0.";
+    }
+  }
+  if (operation === '%') {
+    return one.mod(two).toString();
+  }
+  throw Error(`Unknown operation '${operation}'`);
```

Comment:
- [ ] When I am performing an operation I am able to see the first number only let us say `8/2` I only see `8` and then I just see the answer I recommend that you display on the screen all the operation for a better user XP